packages/client/src/v2-events/features/events/AdvancedSearch/TabSearch.tsx

```diff
@@ -56,14 +60,16 @@ function getSectionFields(
   const advancedSearchSections = event.advancedSearch
   const allUniqueFields = getAllUniqueFields(event)
   return advancedSearchSections.map((section) => {
+    const defaultFields = getDefaultSearchFields(section)
     const advancedSearchFieldId = section.fields.map(
       (f: { fieldId: string }) => f.fieldId
     )
```

Comment:
Not a part of your PR but lets improve this a bit!

```suggestion
    const advancedSearchFieldId = section.fields.map((f) => f.fieldId)
```